public/css/themes/modern/_modern.scss

```diff
@@ -1,4 +1,4 @@
-@import "../../lib/bootstrap-social";
```

Comment:
The reference here actually needs to point ot bootstrap social, and needs to be changed to `@import "node_modules/@ladjs/bootstrap-social/bootstrap-social.scss";`  that would be consistent with the way the other themes are.

For switching to different themes from default, you need to change two lines in main.scss:

- The variables: https://github.com/sahat/hackathon-starter/blob/master/public/css/main.scss#L1
- The scsss: https://github.com/sahat/hackathon-starter/blob/master/public/css/main.scss#L10